tests/ltp/Makefile

```diff
@@ -159,11 +179,12 @@ endif
 
 ifeq ($(FS),ramfs)
 one:
-	rm -rf ramfs.appdir
+	sudo rm -rf ramfs.appdir
 	mkdir -p $(shell dirname ramfs.appdir/$(TEST))
 	cp appdir/$(TEST) ramfs.appdir/$(TEST)
+	cp -r appdir/etc  ramfs.appdir/
 	$(MYST) mkcpio ramfs.appdir ramfs
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) $(FS) $(TEST) $(NL)
+	sudo -E $(RUNTEST) $(MYST_EXEC) $(OPTS) $(FS) $(TEST) $(NL)
```

Comment:
because of sockets! I am trying uid mapping to see if that is sufficient

Reply:
mapping is not sufficient. LTP tests expect to run as root. Any time we call out it needs to be root otherwise various things will fail. These tests probably failed before because setting identity was not supported.